openhcl/openhcl_boot/src/main.rs

```diff
@@ -276,15 +277,17 @@ fn shim_parameters(shim_params_raw_offset: isize) -> ShimParams {
 /// The maximum number of reserved memory ranges that we might use.
 ///
 /// 1. VTL2 parameter regions (could be up to 2).
-/// 2. Sidecar image.
-/// 3. One reserved range per sidecar node.
-pub const MAX_RESERVED_MEM_RANGES: usize = 3 + sidecar_defs::MAX_NODES;
+/// 2. Preserved DMA buffers and hardware queues.
+/// 3. Sidecar image.
+/// 4. One reserved range per sidecar node.
```

Comment:
style: can you move these various bullet points to just be doc comments on the particular enum itself?

Reply:
(and then update the docs for this constant to point to the ResevedMemoryType enum)

Reply:
ok